ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+Hello! Thanks for contributing. 
+
+If you are opening an issue about the Maki editor, style guidelines or another Maki-related component, please provide a detailed description and any supporting imagery. If you are making an icon request, please make sure the following items are in your issue. 
+
+#### Icon Request
+- [ ] Make the issue title: "[icon name] icon." Bad: "request for a garden icon." Good: "garden icon." 
```

Comment:
> Bad: "request for a garden icon." Good: "garden icon." 

Instead maybe:

> E.g. "garden icon" or "place-of-worship icon"

^ Avoids saying that anything is "bad", and gives an example of a multi-word name.